services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py

```diff
@@ -58,11 +76,15 @@ async def _task_docker_compose_up(
     message = f"Finished {command} with output\n{stdout}"
 
     if finished_without_errors:
+        await _send_message(rabbitmq, "service containers started")
```

Comment:
I am not waiting for answers. The message gets put in a queue and from there it is sent in batches at a regular time interval.
If I were to use the non async version of the `queue.put_nowait` it would not guarantee the message delivery order.